bin/main.rb

```diff
@@ -1 +1,63 @@
-puts 'Hello World!'
+puts "Welcome to Pascal and Hillary's tic-tac-toe game"
```

Comment:
/you are missing the line to make this file executable.
`#!/usr/bin/env ruby` needs to be added and the right permissions need to be added. Please see this [screenshot](requirements_screenshots)